src/deltacode/__init__.py

```diff
@@ -80,6 +81,27 @@ def align_scans(self):
             for f in self.old.files:
                 f.original_path = f.path
 
+    def similarity(self):
+        # new_index = self.new.index_files()
+        # old_index = self.old.index_files()
+        # for path, new_files in new_index.items():
+        #     for new_file in new_files:
+        #         for path, old_files in old_index.items():
+        #             for old_file in old_files:
+        for delta in self.deltas:
+            new_fingerprint = delta.new_file.fingerprint
+            old_fingerprint = delta.old_file.fingerprint
+            if new_fingerprint != None and old_fingerprint != None:
+                new_fingerprint = utils.bitarray_from_hex(delta.new_file.fingerprint)
+                old_fingerprint = utils.bitarray_from_hex(delta.old_file.fingerprint)
+                hamming_distance = utils.hamming_distance(new_fingerprint, old_fingerprint)
+                if hamming_distance >= 0 :
```

Comment:
For the items with a `hamming_distance` > 0, we will need to adjust the `Delta.score` field in the Delta objects. The larger the hamming distance, the greater the score value should be added.